pipeline_crab.py

```diff
@@ -71,30 +82,35 @@
                                 'timing_width': 6,
                                 'central_sample': 11}
 
-    peak_position = utils.fake_timing_hist(time_integration_options['n_samples'],
-                                           time_integration_options['timing_width'],
-                                           time_integration_options['central_sample'])
-
-    time_integration_options['peak'], time_integration_options['mask'], time_integration_options['mask_edges'] = \
-        utils.generate_timing_mask(time_integration_options['window_start'], time_integration_options['window_width'],
-                                   peak_position)
+    peak_position = utils.fake_timing_hist(
+        time_integration_options['n_samples'],
+        time_integration_options['timing_width'],
+        time_integration_options['central_sample'])
+
+    (
+        time_integration_options['peak'],
+        time_integration_options['mask'],
+        time_integration_options['mask_edges']
+    ) = utils.generate_timing_mask(
+        time_integration_options['window_start'],
+        time_integration_options['window_width'],
+        peak_position
+    )
 
     # Image cleaning configuration
```

Comment:
as these are image cleaning parameters maybe it is better to set them with command line

Reply:
Yep, can be done .. not in this PR. This PR tries to make this script runnable.